.github/workflows/testing-and-deployment.yml

```
@@ -37,7 +37,7 @@ jobs:
 
       - name: Install package and smoke test
         run: |
-          pip install pyiges --disable-pip-version-check --find-links=dist 
+          pip install pyiges --disable-pip-version-check --find-links=dist
```

This should fix the issue

```suggestion
          pip install -e . --disable-pip-version-check --find-links=dist
```